daemons/fenced/fenced_commands.c

```diff
@@ -2010,7 +2030,7 @@ add_action_specific_attributes(xmlNode *xml, const char *action,
         crm_xml_add_int(xml, F_STONITH_DELAY_MAX, delay_max / 1000);
     }
 
-    delay_base = get_action_delay_base(device, action);
+    delay_base = get_action_delay_base(device, action, NULL);
```

Comment:
We have the victim when calling add_action_specific_attributes(), so we can add it as an argument and pass it here